frontend/src/app/settings/modelsources/t4c-settings/t4c-instance-settings/t4c-repo-deletion-dialog/t4c-repo-deletion-dialog.component.html

```diff
@@ -6,14 +6,41 @@
 <h1 mat-dialog-title>Remove T4C repository</h1>
 <div mat-dialog-content>
   <p>
-    Do you really want delete the T4C repository {{ repo.name }} with the ID
-    {{ repo.id }}?
+    Do you really want to delete the T4C repository
+    <span class="monospace">{{ repo.name }}</span
+    >? <br />
+    This will delete the repository in our database AND on the TeamForCapella
+    server.
   </p>
-  <p>The deletion cannot be undone!</p>
-</div>
-<div mat-dialog-actions>
-  <button mat-button (click)="this.dialogRef.close(false)">No</button>
-  <button mat-flat-button (click)="remoteRepository()">
-    Remove the repository
-  </button>
+  <p><b>The deletion cannot be undone!</b></p>
 </div>
+Please type in the name of the repository to confirm the deletion: <br />
+<form>
+  <mat-form-field appearance="fill">
+    <mat-label>Repository name</mat-label>
+    <input
+      (paste)="(false)"
```

Comment:
Why preventing the paste action? This still require a focused human action for every repository to be deleted but can help when the names are too long.

Reply:
When I'm allowed to paste, I can copy&paste faster than I read the dialog. This is exactly what I'd like to prevent with the dialog. Deleting a TeamForCapella repository has extensive impacts.